frontend/templates/views/partials/component-intro.j2

```diff
@@ -43,13 +43,20 @@
   <div class="ap-o-component-intro-references-container">
     <div class="ap-o-component-intro-box ap-o-component-intro-box-reference">
       <h4>Examples</h4>
+      <input type="checkbox" id="load-examples-checkbox"/>
       <div class="ap-o-component-intro-box-examples">
         {% for example in examples[doc.title] %}
         [% if format in {{ example.formats|tojson }} %]
         <a href="{{ example.url }}">{{ example.title }}</a>
         [% endif %]
         {% endfor %}
       </div>
+      {% if examples[doc.title]|length > 6 %}
+      <label class="ap-o-component-intro-load-btn" for="load-examples-checkbox">
```

Comment:
Suggestion: use the `toggleClass` action instead as it makes this easier and the markup clearer.